csharp/nuget/update.sh

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# To update the required dependencies, follow these steps:
+# 1. Add the desired nuget packages into the 'paket.dependencies' file.
+# 2. Run this shell script !from this directory!: ./update.sh.
+# 3. Check the fetched packages in the regenerated 'paket.csharp_deps.bzl' file.
+# 4. Reference this packages in your BUILD files via '@paket.csharp_deps//{package.name.in.lowercase}'.
+
+paket install
```

Comment:
So this means that CSharp build is _not_ hermitic right? We should outline this in the PR description and note the build requirements on the local system.

Reply:
Well, it depends on what we consider as hermetic. You're right that this file is not hermetic (I forgot about it), but it is needed only if you want to run `paket2bazel` (read as "run `update.sh`"), which is needed only for developers when you update dependencies. The driver and its build afterwards is hermetic and I've already "tested" it while setting up my Linux env: I just needed to install Bazel and fetch the repo.

Reply:
I see - build is hermitic, but updating dependencies is not. Correct?

Reply:
Correct.